sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -30,7 +30,7 @@
     <Compile Include="$(AzureCoreSharedSources)ConditionalRequestOptionsExtensions.cs" />
     <Compile Include="$(AzureCoreSharedSources)DiagnosticScope.cs" />
     <Compile Include="$(AzureCoreSharedSources)HashCodeBuilder.cs" />
-    <Compile Include="$(AzureCoreSharedSources)NoBodyResponse{T}.cs" />
```

Comment:
where these names causing issues? or why the rename?

Reply:
Files that have `{` or `}` in the name cause a crash in the coverage library we use. We filed an issue but until it's fixed and released we have to rename our source files.

Reply:
Thank you for the context; that addresses my question below as well.

Reply:
Is there a place where this can be documented so until the bug is fixed, other people in the repo can avoid it?

Reply:
@jsquire keep in mind as well the class name doesn't change. Eliding `{T}` from a file name is pretty common. SA1649 accepts with and without. We have many generic classes without them currently.